docs/reference/setup/install/docker-compose.yml

```
@@ -66,7 +66,7 @@ services:
     depends_on:
       setup:
         condition: service_healthy
-    image: {docker-repo}:{version}
+    image: {docker-repo}:${STACK_VERSION}
```

If someone downloads this file from GitHub ([which we say is supported](https://www.elastic.co/guide/en/elasticsearch/reference/8.0/docker.html#_prepare_the_environment)), `{docker-repo}` won't expand correctly.
```suggestion
    image: docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION}
```

---

Good point! I'm 👍 to this change, let's see if @lockewritesdocs has feedback since I think he introduced the placeholder.